### PR TITLE
Stocks data

### PR DESCRIPTION
**Pull request by Busiwa24**
**In the following PR I completed these tasks:** 
- Setup the files needed to create the Stocks Data Financial modeling prep API
- I Used React documentation.
- Used React components, props, and router
- Connected React and Redux.
- Handle events in a React app.
- Wrote integration tests with a React testing library.
- Used  styles in a React app as  well as react life cycle methods 
- Applied React best practices and language style guides in code.
- Recorded a short loom video showcasing my projects as per the requirements: 
[]https://www.loom.com/share/454320f2828c4e8986ca253e20e04e8e

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/__tests__/company.test.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import React from 'react';
```
Comment (mikenath223) [inline src/components/__tests__/company.test.js]: - [x] A couple of tests are failing. Kindly check and fix them.

Comment (Busiwa24) [reply src/components/__tests__/company.test.js]: Sorted it out

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @Busiwa24, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**